app/components/Header/Header.jsx

```diff
@@ -45,6 +63,11 @@ export default class Header extends React.Component {
         <div className="inner">
           <h1 className="title">FakeFlix</h1>
           <div className="header-right">
+            <a className="header-cart-container" href={'/cart/'}>
```

Comment:
I think it might look nicer with the Cart icon and count as the right most element in the header